homeassistant/components/eight_sleep/sensor.py

```diff
@@ -295,9 +270,8 @@ def __init__(
         super().__init__(coordinator, eight, None, sensor, units)
 
         self._attr_icon = "mdi:thermometer"
-        self._attr_native_unit_of_measurement: str = (
-            TEMP_CELSIUS if self._units == "si" else TEMP_FAHRENHEIT
-        )
+        self._attr_device_class = SensorDeviceClass.TEMPERATURE
+        self._attr_native_unit_of_measurement = TEMP_CELSIUS
```

Comment:
Could these three be moved to class attributes instead of instance attributes?

```python
class EightRoomSensor(EightSleepBaseEntity, SensorEntity):
    """Representation of an eight sleep room sensor."""

    _attr_icon = "mdi:thermometer"
    _attr_device_class = SensorDeviceClass.TEMPERATURE
    _attr_native_unit_of_measurement = TEMP_CELSIUS

    def __init__(
```